{"name":"GetAllByManager: Failure","fullName":"TestPublicationPgRepoSuiteRunner/PublicationPgRepoSuit/TestPublicationPgRepo_GetAllByManagerFailure","status":"passed","statusDetails":{"message":"","trace":""},"start":1727636171878,"stop":1727636171878,"uuid":"7ec385ac-7e94-11ef-b91f-be2bab8c44a1","historyId":"80ba6367518356e8ab787787f22bbd30","testCaseId":"10b0c4bb9d9329cea16cee2ebbbdeb0a","labels":[{"name":"language","value":"go1.23.1"},{"name":"framework","value":"Allure-Go@v0.6.0"},{"name":"host","value":"MacBook-Pro-Pavel-2.local"},{"name":"thread","value":"TestPublicationPgRepoSuiteRunner/PublicationPgRepoSuit/TestPublicationPgRepo_GetAllByManagerFailure"},{"name":"suite","value":"PublicationPgRepoSuit"},{"name":"package","value":"cookdroogers/internal/repo/pg"},{"name":"tag","value":"PublicationPgRepo"}],"steps":[{"name":"Failure","status":"passed","start":1727636171878,"stop":1727636171878,"steps":[{"name":"ASSERT: Nil","status":"passed","start":1727636171878,"stop":1727636171878,"parameters":[{"name":"Actual","value":"[]models.Publication([]models.Publication(nil))"}]},{"name":"ASSERT: Error Is","status":"passed","start":1727636171878,"stop":1727636171878,"parameters":[{"name":"Error","value":"sql: connection is already closed: error in postgres repo"},{"name":"Target","value":"error in postgres repo"}]}]}]}